### PR TITLE
clone repo along with openpoints

### PR DESCRIPTION
**Pull request by jaswanthbjk**
## What does this PR do?
Modify README to clone the repo along with openpoints which is a dependency

Fixes #\<issue_number>

## Before submitting

- [x] Did you make sure **title is self-explanatory** and **the description concisely explains the PR**?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [x] Did you **test your PR locally** with `pytest` command?
- [x] Did you **run pre-commit hooks** with `pre-commit run -a` command?

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Please see the [Project Guidelines](#project-guidelines) section before diving i
 
 ```bash
 # clone project
-git clone https://github.com/CCInc/3d-ml.git && cd 3d-ml
+git clone https://github.com/CCInc/3d-ml.git --recursive && cd 3d-ml
```
Comment (CCInc) [inline README.md:20]:
```suggestion
git clone https://github.com/CCInc/3d-ml.git --recurse-submodules && cd 3d-ml
```

**Review by CCInc — COMMENTED** [review]
Thanks for the PR! Looks like the --recursive argument might be deprecated, let me know if this works instead!

**Comment by leo-stan** [comment]
https://git-scm.com/book/en/v2/Git-Tools-Submodules#:~:text=If%20you%20pass%20%2D%2Drecurse,the%20repository%20have%20submodules%20themselves.
Yep seems that `--recurse-submodules` is the way to go :)

**Comment by leo-stan** [comment]
@jaswanthbjk thanks for your contribution! If you have any more time to give to the project feel free to reach out :)

**Comment by jaswanthbjk** [comment]
@leo-stan thanks for the merge. I am already in contact with @CCInc and will be starting to contribute at a slower pace.

**Pull request merged**